src/main/java/de/rabea/Game/Board.java

```diff
@@ -37,6 +51,26 @@ public boolean isPositionAvailable(int position) {
                 !(cells[position] == Cell.X || cells[position] == Cell.O);
     }
 
+    public Mark switchMark(Mark mark) {
+        if (mark == Mark.X) {
```

Comment:
You could statically import Mark, then the code would read 
`if (mark == X)` which is quite nice. 

You can do this with intelliJ, alt-enter and select 'statically import' option